docs/glyph-inspector.html

```diff
@@ -130,6 +130,7 @@ <h1>Free Software</h1>
     var glyph = font.glyphs.get(glyphIndex),
         html = '<dl>';
     html += '<dt>name</dt><dd>'+glyph.name+'</dd>';
+    html += '<dt>glyph</dt><dd>'+String.fromCodePoint.apply(null, glyph.unicodes) + '</dd>';
```

Comment:
When there are multiple code points for a glyph, all characters are just concatenated. A comma separated list would be preferrable. Also "glyph" is not really the correct term as a label here, because it's not the glyph we're showing, but readable characters for codepoints that are matched to that glyph.